packages/components/psammead-navigation/src/index.jsx

```diff
@@ -90,6 +80,44 @@ const StyledLink = styled.a`
   }
 `;
 
+const StyledListItem = styled.li`
+  display: inline-block;
+  position: relative;
+  z-index: 2;
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
```

Comment:
![Screenshot 2019-11-21 at 14 11 09](https://user-images.githubusercontent.com/6690955/69346050-0d741700-0c6a-11ea-9191-847048644d6d.png)

This doesn't quite look right - would it be easier to keep the hover/focus states as they are, and just add the extra padding to the the list element instead of the link? Or the whole `<ul>` if that works?

Reply:
Using `margin` instead of `padding` solves the issue :)

Reply:
Ok cool! :) Could you just do
```
&:last-child {
      ${({ dir }) => css`
        margin-${dir === 'ltr' ? 'right' : 'left'}: ${GRADIENT_WIDTH}; 
  `}
}
```
and not have to override StyledLink styles at all?